components/atoms/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link'
+import PropTypes from 'prop-types'
+import styles from './Breadcrumbs.module.css'
+
+/**
+ * Render the Breadcrumbs component.
+ *
+ * @author WebDevStudios
+ * @param {object} props             The component attributes as props.
+ * @param {Array}  props.breadcrumbs The breadcrumb array.
+ * @return {Element}                 The Breadcrumbs component.
+ */
+export default function Breadcrumbs({breadcrumbs}) {
+  return (
+    <>
+      {!!breadcrumbs && !!breadcrumbs.length && (
```

Comment:
@dcooney Thank you for the PR. Got a possible improvement...

```suggestion
      {!!breadcrumbs?.length && (
```